specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/customer.json

```diff
@@ -75,214 +83,246 @@
+            "$ref": "./examples/customersListByBillingAccount.json"
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
         }
       }
     }
   },
   "definitions": {
     "Customer": {
+      "description": "A partner's customer.",
+      "type": "object",
       "allOf": [
         {
           "$ref": "./types.json#/definitions/ArmResource"
         }
       ],
-      "description": "A partner's customer.",
       "properties": {
         "properties": {
-          "$ref": "#/definitions/CustomerProperties",
-          "description": "The customer.",
+          "description": "The properties of a(n) Customer",
+          "allOf": [
+            {
+              "$ref": "#/definitions/CustomerProperties"
+            }
+          ],
           "x-ms-client-flatten": true
         }
       },
-      "type": "object"
+      "x-ms-azure-resource": true
     },
     "CustomerListResult": {
-      "description": "The list of customers.",
+      "description": "A container for a list of resources",
+      "type": "object",
       "properties": {
         "nextLink": {
           "description": "The link (url) to the next page of results.",
-          "readOnly": true,
-          "type": "string"
-        },
-        "totalCount": {
-          "description": "Total number of records.",
-          "format": "int32",
-          "readOnly": true,
-          "type": "number"
+          "type": "string",
+          "readOnly": true
         },
         "value": {
-          "description": "The list of customers.",
+          "description": "The list of resources.",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/Customer"
           },
           "readOnly": true,
-          "type": "array"
+          "x-ms-identifiers": [
+            "id"
+          ]
         }
       }
     },
     "CustomerProperties": {
-      "description": "The properties of a customer.",
+      "description": "A partner's customer.",
+      "type": "object",
       "properties": {
         "billingProfileDisplayName": {
-          "description": "The name of the billing profile for the invoice section.",
-          "readOnly": true,
-          "type": "string"
+          "description": "The name of the billing profile for the invoice section or customer.",
+          "type": "string",
+          "readOnly": true
         },
         "billingProfileId": {
-          "description": "The ID of the billing profile for the invoice section.",
-          "readOnly": true,
-          "type": "string"
+          "description": "The fully qualified ID that uniquely identifies a billing profile.",
+          "type": "string",
+          "readOnly": true
         },
         "displayName": {
           "description": "The name of the customer.",
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "enabledAzurePlans": {
```

Comment:
Can we remove "enabledAzurePlans" from our new GA APIs? This is a concept we're trying to deprecate. Subscription RP will be creating rating assets if they don't already exist. There should be no reason for our customers to know about this going forward.

Reply:
@ramkd12 @amberbhargava to comment here

Reply:
Removing enabledAzurePlans in favor of `isDevTestEnabled` has a cost associated with it. We will evaluate and make a decision based on cost.

Reply:
We could make a small change to remove `enabledAzurePlans` in favor of `isDevTestEnabled` with the following change.

```python
service always enables Sku 001
if isDevTestEnabled
  service enables Sku 002

If we want the granularity to only enable Sku 002, then this cannot be represented with a single boolean.

However, there would be no service level changes required for this update.

Reply:
While we agree with the direction, partners will need to make changes as well and it's better that this change is driven as a feature rather than part of the GA.